exporter/fileexporter/README.md

```diff
@@ -20,16 +23,29 @@ This intended for primarily for debugging Collector without setting up backends.
 The following settings are required:
 
 - `path` (no default): where to write information.
+- `rolling_logger_options` (default): an option to self-rotate log files.
+    - max_size:   the maximum size in megabytes of the log file before it gets rotated. It defaults to 100 megabytes.
+    - max_age:   the maximum number of days to retain old log files based on the  timestamp encoded in their filename.The default is not to remove old log files based on age.
+    - max_backups:  the maximum number of old log files to retain.  The default is to retain all old log files
```

Comment:
```suggestion
    - max_backups: [no default (unlimited)]: the maximum number of old log files to retain.
```

Reply:
I think possibly we should include a default value here, something high enough that it does not easily affect users, but low enough that it will not crash most hosts. Maybe 100, so we have effectively a max of 10GB for all files. What do you think?

Reply:
I think It's a great idea.